ansible_collections/arista/avd/extensions/molecule/eos_designs-twodc-5stage-clos/digital_twin/intended/structured_configs/DC1-POD1-L2LEAF1A.yml

```diff
@@ -32,7 +32,6 @@ local_users:
   privilege: 15
   role: network-admin
   sha512_password: $6$eJ5TvI8oru5i9e8G$R1X/SbtGTk9xoEHEBQASc7SC2nHYmi.crVgp2pXuCXwxsXEA81e4E0cXgQ6kX08fIeQzauqhv2kS.RGJFCon5/
-  no_password: true
```

Comment:
this is removed because sha512 will take precedence in eos cli config gen template so not needed.